zip-codes/scripts/map.js

```diff
@@ -1,23 +1,32 @@
 'use strict';
 
-function initMap() {
+function initMap(newMap) {
   // Create a map object and specify the DOM element for display.
   var map = new google.maps.Map(document.getElementById('map'), {
-    center: {lat: 47.611435, lng: -122.330456},
+    center: newMap,
     scrollwheel: true,
     zoom: 8
   });
-  if (zips.length) {
-    // zips.val() 
-    console.log(zips.val());
-  }
-  var markerCity = {lat:$(this) ,lng:$(this)};
+
   var marker = new google.maps.Marker({
-    position: 'Seattle',
+    position: newMap,
```

Comment:
Perfect! Nice work passing your lat/lng data to the initMap function and using it to position the marker :)